webapp/Login/LoginForm/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { useDispatch, useSelector } from 'react-redux'
+import * as R from 'ramda'
+
+import Error from '../Error'
+
+import { resetPassword, resetPasswordFormReset } from '../../actions'
+
+const ForgotPassword = (props) => {
+  const { onClose } = props
+
+  const dispatch = useDispatch()
+  const emailRef = useRef(null)
+  const { error, message } = useSelector(R.pathOr({}, ['login', 'localLogin', 'resetPassword']))
```

Comment:
possible to use LoginState.getResetPassword or similar?

Reply:
there's another issue on the board. I mentioned this morning I wouldn't refactor the redux store in this issue